README.md

```diff
@@ -160,10 +160,13 @@ By default the size of the container (ram, disk, cpu shares) uses the package in
 the internal "sdc_" set of packages closest to 'ram=1024 MiB'. The "sdc_"
 packages are really only applicable for development. More appropriate for
 production is a set of packages separate from "sdc_". The following can be
-run to add a number of "t4-*" packages and to configure the Docker service
+run from the global zone to add a number of "t4-*" packages and to configure the Docker service
 to use them:
 
-    sdc-login docker /opt/smartdc/docker/tools/gen_packages.js
+
+    papi_svc=$(sdc-sapi /services?name=papi | json -Ha uuid)
+    sdc-sapi /services/$papi_svc -X PUT -d '{ "metadata": { "IGNORE_CPU_CAP": true } }'
+    ./tools/gen_packages.js
```

Comment:
ah, IGNORE_CPU_CAP yah. I'm thinking we should just change this to 'sdcadm post-setup dev-sample-data' to get some sample packages.